content/oddnews.njk

```diff
@@ -21,12 +24,13 @@ summary: |
 {%- set title = 'What to Expect' -%}
 {%- set contents -%}
   Here are a few examples of topics featured in past editions of OddNews.
-  - **Bird of the Month** - Introduction to the oddest of birds
   - **CSS** - [Miriam Suzanne's](/authors/miriam/) latest work in Container Queries
-  - **Python** - How to Run Playwright Inside Docker
   - **Sass** - How to use the new `split()` function in Sass
-  - **Custom App Development** - Web Apps vs. PWAs vs. Native
-  - **UX Design** - How to conenct designers and developers with Object Oriented UX
+  - **JavaScript** - Guide to the new Node.js package importer
```

Comment:
```suggestion
  - **JavaScript** - Guide to Sass's Node.js package importer
```
edited @jgerigmeyer's suggestion. Maybe we want to keep new in though and check theses every few months?